Lib/fnmatch.py

```diff
@@ -143,50 +150,33 @@ def _translate(pat, STAR, QUESTION_MARK):
                     # Negated empty range: match any character.
                     add('.')
                 else:
+                    # Escape set operations (&&, ~~ and ||).
+                    stuff = _set_ops_re.sub(r'\\\1', stuff)
```

Comment:
I could also cache the call to `sub` as I said to Serhiy in another PR. Will do it tomorrow.